pydsa/__init__.py

```diff
@@ -7,3 +7,5 @@
 from .sleep_sort import sleep_sort
 from .bfs import bfs
 from .binary_search import binary_search
+from .linear_search import linear_search
+from .queue import *
```

Comment:
You should not use `from module import *`

Please use here proper methods which you wish to import